### DISCUSSION TITLE
Does the Docker contain Langroid and Examples?

### DISCUSSION THREAD

**Discussion by 99bits** [langroid/langroid#312] [Q&A]
Thank you for this amazing project.  
We would like to test langroid together with examples.  Does the Docker contain everything needed or do we need to setup langroid first and then also the examples separately.

**Comment by pchalasani** [comment]
Thanks for trying out Langroid. @Mohannadcse  is our docker expert, I will have him respond.

**Comment by Mohannadcse** [comment]
Thanks @99bits, for reaching out. I'm working on some enhancements for the docker setup and will push them soon, and will let you know.

**Comment by Mohannadcse** [comment]
@99bits we have pushed the enhancements. The docker image that will be pulled contains all the needed files and setup. Just please follow the instructions described [here](https://github.com/langroid/langroid#whale-docker-instructions). Please let us know if you face any issues.

**Comment by pchalasani** [comment]
@Mohannadcse can you specifically answer the question about whether it contains examples as well? thanks

**Comment by Mohannadcse** [comment]
The examples will be inside the docker image. So once you execute the docker run command in the instructions, you can go to the directory `examples`. It should look like this, you can start with the examples inside the directory `basic`.

<img width="741" alt="image" src="https://github.com/langroid/langroid/assets/15859139/872d94b8-d1aa-4ea2-be96-f771a65ddb19">

**Comment by Mohannadcse** [comment]
I think the problem is solved. @99bits feel free to re-open it if you still face any problem.
